frontend/src/components/AddDebuggingSessionModal.vue

```diff
@@ -19,22 +51,19 @@ interface Props {
 
 const props = defineProps<Props>();
 
-const datasets = ref<DatasetDTO[]>([]);
-const models = ref<ModelDTO[]>([]);
 const currentStep = ref(1);
 const loading = ref(false);
-
 const dialog = ref(false);
 const sessionName = ref("");
-const selectedDataset = ref<DatasetDTO | null>(null);
-const selectedModel = ref<ModelDTO | null>(null);
+const selectedDatasetId = ref<string | null>();
+const selectedModelId = ref<string | null>();
 
 const isMLWorkerConnected = computed(() => {
   return state.workerStatus.connected;
 });
 
 const missingValues = computed(() => {
-  if (selectedDataset.value === null || selectedModel.value === null) {
+  if (selectedDatasetId.value === null || selectedModelId.value === null) {
     return true;
   }
   return false;
```

Comment:
Can be simplified to: `return selectedDatasetId.value === null || selectedModelId.value === null`